migrations/2024-11-21-195004_add_report_count/up.sql

```diff
@@ -43,6 +56,47 @@ FROM (
 WHERE
     a.post_id = cnt.post_id;
 
+-- Re-enable triggers after upserts
+ALTER TABLE post_aggregates ENABLE TRIGGER ALL;
+
+-- Re-enable indexes
+UPDATE
+    pg_index
+SET
+    indisready = TRUE
+WHERE
+    indrelid = (
+        SELECT
+            oid
+        FROM
+            pg_class
+        WHERE
+            relname = 'post_aggregates');
+
+-- reindex
+REINDEX TABLE post_aggregates;
```

Comment:
This is practically the same as dropping then recreating the index. The only benefit of `REINDEX` is that it doesn't block concurrent reading.